getDeps.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Please rename this `getSDO.sh` because that's what it is doing is getting a new version of Intel's SDO code